VideoLocker/res/values/strings.xml

```diff
@@ -183,7 +183,7 @@
     <!-- Label for the menu popup for viewing the course on the web -->
     <string name="course_view_on_web">View on Web</string>
     <!-- Label for the menu popup for changing mode to Videos Only/Full Course -->
-    <string name="course_change_mode">Mode picker</string>
+    <string name="course_change_mode">Course mode picker</string>
```

Comment:
@BenjiLee Great suggestion: I'd be in favor of making the text in the opened menu match!